bookdown/04-pipelines.Rmd

```diff
@@ -317,41 +322,26 @@ branch_graph = mlr_pipeops$get("branch", c("nop", "pca", "scale")) %>>%
 
 Without "unbranching" one creates the following graph:
 
-```{r 04-pipelines-028, eval=FALSE}
-branch_graph$plot(html=TRUE)
-```
-
-```{r 04-pipelines-029, echo=FALSE, fig.align="right"}
-branch_graph$plot(html=TRUE) %>% visInteraction(zoomView = FALSE) # disable zoom
+```{r 04-pipelines-028}
+graph$plot(html = TRUE) %>%
+  visInteraction(zoomView = FALSE) # disable zoom
 ```
 
 Now when "unbranching", we obtain the following results:
 
-```{r 04-pipelines-030, fig.align="center"}
-(branch_graph %>>% mlr_pipeops$get("unbranch", c("nop", "pca", "scale")))$plot(html = TRUE) %>%
+```{r 04-pipelines-029}
+(graph %>>% mlr_pipeops$get("unbranch", c("nop", "pca", "scale")))$plot(html = TRUE) %>%
   visInteraction(zoomView = FALSE) # disable zoom
 ```
 
-The same can be achieved using a shorter notation:
-
-```{r 04-pipelines-031}
-# List of pipeops
-opts = list(po("nop", "no_op"), po("pca"), po("scale"))
-# List of po ids
-opt_ids = mlr3misc::map_chr(opts, `[[`, "id")
-po("branch", options = opt_ids) %>>%
-  gunion(opts) %>>%
-  po("unbranch", options = opt_ids)
-```
-
```

Comment:
It is a bit hard to understand why this got deleted. Please do not mix various issues in one PR because this makes it hard to review.